frontend/src/components/Psychologist/Appointments/NewAppointment.jsx

```diff
@@ -29,12 +29,18 @@ const NewAppointment = () => {
   }, []);
 
   const createNewAppointment = e => {
-    e.preventDefault();
-    setNotification({});
-    agent.Appointment.add(patientId, date).then(response => {
-      history.push('/psychologue/mes-seances');
-      setNotification(response);
-    });
+    const beginningDate = new Date('2021-03-21');
+    const today = new Date();
+    const todayUTC = convertLocalToUTCDate(today);
+    const diffInMonth = Math.abs(date.getMonth() - todayUTC.getMonth());
+    if (diffInMonth <= 4 || (date >= beginningDate && date <= today)) {
```

Comment:
no ! should be managed in the DatePicker 
https://github.com/Hacker0x01/react-datepicker/blob/master/docs/datepicker.md